### ISSUE TITLE
vim editing mode, can't press escape.

### ISSUE THREAD

**Issue by dessalines** [hackmdio/codimd#1361]
When pressing escape in vim edit mode, it follows the browser escape. How do I exit mode?

**Comment by Yukaii** [comment]
Hi @dessalines , do you have any browser vim extension installed? It might be one reason that causes this.

**Comment by dessalines** [comment]
Nope, just regular old firefox, which treats the escape key as losing focus from the browser window.

**Comment by thor314** [comment]
Just echo-ing this, using flatpak firefox on linux, can't use vim mode as escape goes to browser escape. Have used on other browsers, ctrl-c appears to function as escape as well; but ctrl-c goes to copy in Firefox.

**Comment by LinusSch** [comment]
Echo: this bug also occurs on Vivaldi Flatpak version, no extensions installed, with single key shortcuts disabled.

**Comment by LinusSch** [comment]
### Workaround

From [Vim Tips Wiki - Avoid the escape key](https://vim.fandom.com/wiki/Avoid_the_escape_key) we learn several tricks that do not help on hackmd.io (Ctrl+c, Alt in the terminal) but one thing that does:

`:imap jj <Esc>`

Substitute `jj` for a quick sequence that you do not have to type in your language. Note that hackmd.io does have `:imap`, but not `:inoremap`.
